sample-code/vc-using-framestream/vc-using-framestream/Connection.cpp

```diff
@@ -142,6 +164,8 @@ void Connection::SerializeCommand(const aris::Command & cmd, std::vector<uint8_t
 
 void Connection::SendCommand(const aris::Command & cmd)
 {
+  std::cout << "Connection::SendCommand: " << cmd.DebugString() << '\n';
```

Comment:
I'm assuming that you intended to leave this console logging here.  I think it's useful.